main.go

```diff
@@ -33,6 +42,64 @@ func HandleSignals(ctx context.Context) {
 	os.Exit(code)
 }
 
+func loadCachedStates(states *[]CloudflareState) error {
+	if _, err := os.Stat(cachePath); err == nil {
```

Comment:
To avoid using `if/else` you can do :

```
if err != nil { 
     log.debugf("no cache")
     return nil
}
// do what you want to do
```

cf. https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88